### PR TITLE
Change text on /matching

### PR DESCRIPTION
**Pull request by NathOrtega**
## Initial State
236680353-a0ed85f2-e158-4ae3-8ddc-f31427d27599

`/group` and `/matching` were sharing the same text
https://github.com/corona-school/project-user/issues/717

## Current State
Screenshot 2023-06-19 at 4 18 24 PM

Now the `/matching` page has the updated text, while the `/group` maintains the original text.
In order to make this change I had to edit the `courseTypes` in `src/lang/introduction/de.tsx` and make it an object with two keys: `groupCourses` for the text in `/group` and `oneToOneLearningSupport` for the text in `/matching`.

NOTE 👀
In the figma design we can see an extra button to become a tutor, as well as a version with a message for when that button was clicked, I realized that that logic is already implemented and **I did not** change anything about it.
Screenshot 2023-06-19 at 4 22 45 PM
Screenshot 2023-06-19 at 4 26 27 PM

### CODE REVIEW

File: src/widgets/OnboardingCard.tsx [hunk 1]
```diff
@@ -5,7 +5,8 @@ import RequestInstructorTutorBanner from './RequestInstructorTutorBanner';
 
 type CardProps = {
     headline: string;
-    Description: React.FC;
+    Description: string;
```
Comment (Jonasdoubleyou) [inline src/widgets/OnboardingCard.tsx:8]: (by convention props should be camelCased, i.e. `description` and `bulletPoints`)

File: src/pages/student/onboarding/GroupOnboarding.tsx [hunk 2]
```diff
@@ -65,7 +65,8 @@ const GroupOnboarding: React.FC<OnboardingProps> = ({ canRequest = false, waitFo
                     <Box justifyContent="center" alignItems="center">
                         <OnboardingCard
                             headline={t('introduction.groupCourses')}
-                            Description={waitForSupport ? GroupRequestedInfos : GroupOnboardingInfos}
+                            Description={t('introduction.courseTypes.groupCourses')}
+                            BulletPoints={waitForSupport ? GroupRequestedInfos : GroupOnboardingInfos}
```
Comment (Jonasdoubleyou) [inline src/pages/student/onboarding/GroupOnboarding.tsx:69]: I think we also need different text for the bulletpoints, the text currently doesn't make much sense

File: src/widgets/OnboardingCard.tsx (hunk 1 above)
Comment (NathOrtega) [reply src/widgets/OnboardingCard.tsx]: Sorry, I did not see that 🫣😳 I've already fixed it.

File: src/pages/student/onboarding/GroupOnboarding.tsx (hunk 2 above)
Comment (NathOrtega) [reply src/pages/student/onboarding/GroupOnboarding.tsx]: Could you please tell me who should I ask about a new text? Or could you please tell me where can I find an updated version of it, so I can replace it? Thanks! 😊

Comment (Jonasdoubleyou) [reply src/pages/student/onboarding/GroupOnboarding.tsx]: I think there is a new text in the linked Figma file in the issue ...

Comment (Jonasdoubleyou) [reply src/pages/student/onboarding/GroupOnboarding.tsx]: image
image

here's a screenshot of the different texts

Comment (Jonasdoubleyou) [reply src/pages/student/onboarding/GroupOnboarding.tsx]: Sorry, I'm not quite sure what I saw there when I reviewed, I just checked it and all the texts seem to be correct ...

**Review by Jonasdoubleyou — APPROVED** [review]

**Pull request merged**